runtime/local/src/lib.rs

```diff
@@ -268,12 +269,35 @@ impl OnUnbalanced<NegativeImbalance> for OnBlockReward {
 
 parameter_types! {
     pub const RewardAmount: Balance = 2_664 * MILLIAST;
+    pub const DAppsRewardPercentage: u32 = 50;
 }
 
 impl pallet_block_reward::Config for Runtime {
     type Currency = Balances;
     type OnBlockReward = OnBlockReward;
     type RewardAmount = RewardAmount;
+    type DAppsRewardPercentage = DAppsRewardPercentage;
```

Comment:
It’s unclear why this pallet modification required.

Reply:
The current implementation of `pallet_block_reward` is using raw number to divide rewards for maintenance and `pallet_dapp_staking`.
The idea behind this is to use same const percantage in `pallet_block_reward` and pallet-`dapps`-staking to avoid possible mistakes in calculation